README.md

```diff
@@ -34,8 +34,8 @@ console.log('hello, world!')
 
 # Learning More
 
-* Read the [tutorial](http://sweetjs.org/doc/1.0/tutorial.html).
-* Read the [reference documentation](http://sweetjs.org/doc/1.0/reference.html).
+* Read the [tutorial](https://www.sweetjs.org/doc/tutorial.html).
+* Read the [reference documentation](https://www.sweetjs.org/doc/reference).
```

Comment:
Could you change the protocol on the editor link as well?

I don't think we need to remove the file extension. If we do, it should be done consistently.

@disnet?